### ISSUE TITLE
add a consciousness feature

### ISSUE THREAD

**Issue by ballsman12** [massclown/HalfSwordTrainerMod#6]
i got decapitated but could resurrect but it keeps passing out (cuz he has no head) itd be really funny if i could walk around with no head

**Comment by massclown** [comment]
Yes, unfortunately, this is a known issue at the moment (see https://github.com/massclown/HalfSwordTrainerMod?tab=readme-ov-file#resurrect-the-player): 
>
> Having exploded head or being decapitated still appears to be fatal in the current version, as are some other injuries.
> 
If/when I figure out how to correctly freeze the health and consciousness at a good level (without crashing the game even more), I will add that!

**Comment by jon007a** [comment]
the mod works but the e/q hotkeys for picking up items do not work

**Comment by massclown** [comment]
> the mod works but the e/q hotkeys for picking up items do not work

You probably have the Split Screen mod enabled, it remaps Q/E to controller, disable it when you don't need it.

**Comment by Sch1t** [comment]
Would it be possible to spawn add-on weapons through mods? Like using OVRDTHS Selectors Weapon packs and spawning those weapons with the menu.
